ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used:
+* Browser Name and version:
+* Operating System and version (desktop or mobile):
+* Link to your project:
+
+## Context
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+<!--- How has this bug affected you? What were you trying to accomplish? -->
+<!--- If you have errors or a stacktrace in the console, here is a good place
+to put it. Please consider using a pastebin service or create a gist.-->
+<!--- Some screenshots are always greatly appreciated here! -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+
+## Possible Cause/Fix
+<!--- Not obligatory, but suggest a cause or reason for the bug -->
+<!--- Do you have a fix? Why not open a Pull Request then? :-) -->
```

Comment:
It was intended as an encouraging "go ahead, you can do it!", but apparently fails at it. What if I rephrase this way?

> Have a fix? Please open a PR and become an iTowns developer!